utils/getMultipleCountries.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const numberFormat = require('./numberFormat');
+const exitCountry = require('./exitCountry');
+const to = require('await-to-js').default;
+const handleError = require('cli-handle-error');
+
+module.exports = async (spinner, table, states, countryName, options) => {
+	if (countryName && !states && !options.chart) {
+		const [err, response] = await to(
+			axios.get(`https://corona.lmao.ninja/v2/countries/${countryName}`)
+		);
+		exitCountry(err, spinner, countryName);
+		err && spinner.stopAndPersist();
+		handleError(`API is down, try again later.`, err, false);
+		const thisCountry = response.data;
+
+		// Format.
+		const format = numberFormat(options.json);
+
+		thisCountry.map(data => table.push([
```

Comment:
As in line 10, since https://corona.lmao.ninja/v2/countries/${countryName} refers to a single country, has this .map() worked ?

Reply:
I have added a condition here, since previously it worked only for multiple countries input. Now it will work for single country, but the functionality here is for multiple country as mentioned above (https://github.com/ahmadawais/corona-cli/pull/104#discussion_r428179236)